s30899-PatrykSzachniewicz/pamietnik.txt

```diff
@@ -49,3 +49,22 @@ jest jobem (do zapamietania)
 *roznica pomiedzy 'CTRL+C' a 'CTRL+D' (proste)
 *na nastepne cwiczenia przygotowac pytania, na temat niezrozumialych przez nas zagadnien
 
+2023.11.18
+Moj kochany pamietniczku na dzisiejszych zajeciach nauczylem sie:
+*jakie moga byc powody braku mozliwosci utworzenia pliku ns swoim koncie uczelnianym (logiczne)
+*quota, czyli przydzial przestrzeni dyskowej na danego uzytkownika (zakodowane w glowie)
+*w odpowiedziach na zadania trzeba uwazac na literowki bo moga uniewaznic odpowiedz
+*podczas tworzenia zagniezdzonych katalogow trzeba pamietac o parametrze, ktory umozliwi nam to
+*czesto uzywane parametru niektorych polecen trzeba znac na pamiec (zakodowane w glowie)
+*jak przeniesc katalog (do zapamietania)
+*zamykanie programu (przydatne)
+*jak wypisac pliki pasujace do podanego wzorca (trzeba przecwiczyc)
+*wyrazenia regularne (malo zrozumiale)
```

Comment:
Zatem to jest do przećwiczenia. W materiałach były linki do serwisów gdzie można to przećwiczyć.